### ISSUE TITLE
`wmic` is not recognized as an internal or external command, operable program or batch file

### ISSUE THREAD

**Issue by dahlia** [planetarium/9c-launcher#805]
```
[error] Error occurred during initializeHeadless(). Error: Command failed: wmic logicaldisk get size,freespace,caption
'wmic' is not recognized as an internal or external command,
operable program or batch file.
```

Note that [`wmic` is usually placed in *%SystemRoot%\System32\Wbem*][1].  If the path is not listed in the `PATH` environment variable, `wmic` cannot be resolved.  In order to make it more robust, I suggest to invoke `wmic` with the absolute path.

[1]: https://superuser.com/questions/1178674/wmic-is-not-recognized-as-an-internal-or-external-command-operable-program-or

**Comment by BasixKOR** [comment]
This seems like an upstream issue of `check-disk-space` package. Let me see if I could patch it and make it into upstream.

**Comment by BasixKOR** [comment]
Should be resolved by #845. Please reopen if it occurs again.
